### ISSUE TITLE
Documentation: more detailed explanation or an example for auto-config

### ISSUE THREAD

**Issue by KillyMXI** [ota-meshi/eslint-plugin-jsonc#25]
> `plugin:jsonc/auto-config` ... Automatically apply jsonc rules similar to your configured ESLint core rules to JSON.

I find it difficult to understand what stand behind this description. And it's not easy to figure out from the code.

Some example might be helpful.

Update:

I now see that it is about the rules borrowed from ESlint. But this raises another question:
How do I combine JSONC own rules recommended for `*.jsonc` with auto-configured borrowed rules?
Will this work?:

```
{
  "extends": [
    ...,
    "plugin:jsonc/auto-config",
    "plugin:jsonc/recommended-with-jsonc",
    ...
  ]
}
```

It does seem to work. Although I'm not sure whether it is a good way to use it.

**Comment by ota-meshi** [comment]
Hi! @KillyMXI.
Your usage is correct and is what I expected.
Is the document confusing? If so, could you send me a pull request to rewrite the document for clarity?
